src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
```

Comment:
The message 'Bulldozer started working' could be more informative by specifying what type of Machine is working. Consider including the type of machine in the message.